web/styles/app_variables.css

```diff
@@ -582,8 +582,6 @@
        in subscriptions and user group settings overlays */
     --settings-overlay-subheader-height: 3.1429em; /* 44px at 14px/em */
     --settings-overlay-header-button-height: 2em;
-    /* .settings-sticky-footer */
-    --subscriptions-overlay-sticky-footer-height: 60px;
```

Comment:
Interestingly this wasn't being used to set the height of the footers, just to calculate the height of the containers.